Feature/content type

aleesteele opened the pull request:
closes #14 
- Checks if page.collection is spotlights or posts, and renders it accordingly on the page. Styles are in line with those outlined on Zeplin.

jnschrag commented on assets/_sass/components/_content-type.scss:
```diff
@@ -0,0 +1,16 @@
+  }
+
+  .post-block__header &,
+  .post-list & {
```
You can remove this selector. The home page won't use a post-list class once it's finished.

aleesteele replied:
Removed.

aleesteele commented:
@leezaweb Extending the styles from post-type should account for `color__red` and the `600` weight, I believe.

@jnschrag 
- Changed the selector to `post-block` as recommended, because it more general.
- Added the third content type, to account for the Related Posts block.
- I also switched `post-header` (used for the spotlight features) with `post__header`, which is used for the commentaries.
